content/_includes/CLI/AccountGroupDeleteExample.md

```diff
@@ -0,0 +1,17 @@
+---
+---
+
+```
+account group delete id=<GID>
```

Comment:
are the angle brackets required in the command string or are you adding them as part of the variable? If <GUID> is part of the command don't treat it as a variable. If <GUID> is a number, then enter a number variable as a better example.